### ISSUE TITLE
Allow DXF/DWG import

### ISSUE THREAD

**Issue by LyleKeck** [NatLabRockies/floorspace.js#225]
Must have for practitioners.

**Comment by ljbrackney** [comment]
The expectation is that practitioners who need that sort of functionality would use a different workflow - e.g. Sketchup, gbXML import, etc.  The floor plan workflow is intentionally simplified and limited.

**Comment by LyleKeck** [comment]
The [website](http://nrel.github.io/OpenStudio-user-documentation/reference/geometry_editor/#image-import) says DXF may be supported in the future.  I'm just providing my feedback after testing.

**Comment by macumber** [comment]
Thanks for the feedback @LyleKeck we do hear the requests for DXF/DWG and would love to have that feature but there are not a lot web libraries for dealing with those file formats.  So we had to skip this feature for the first release, but hopefully it can be brought in at some point in the future.

**Comment by TheEnergyTutor** [comment]
Yes seriously need this feature.  3-D lighting photometric modeling software is very similar and it allows for import of dwg/dxf.  This is a must have.

**Comment by bgschiller** [comment]
Hi @TheEnergyTutor ,

As far as I know, there's no one actively developing this project. The contract completed and we moved onto different things. I work at Devetry, the consulting co that did most of the development on this project. If you'd like to talk about setting up a new contract, you can email me at brian@devetry.com .

I'm also happy to review/merge PRs, and to try and point you in the right direction if you'd like to attempt the work yourself.
